### ISSUE TITLE
Performing an __in filter queryset no longer works since v5.1

### ISSUE THREAD

**Issue by jgb** [SmileyChris/django-countries#224]
Hello,

something that used to work in v5.0 no longer works in v5.1 and v5.1.1:

In [9]: Account.objects.filter(country='FR')
Out[9]: [<Account: aaa>, <Account: bbb>, <Account: ccc>, '...(remaining elements truncated)...']

In [14]: Account.objects.filter(country__in=['FR'])
Out[14]: []

Downgrading to v5.0 solves the problem.

**Comment by SmileyChris** [comment]
Just did a quick check - this seems to be working fine for me still in 5.1.1 for a basic countryfield at least. Are you using `multiple=True` maybe?

```python
>>> from countries.models import *
>>> Simple.objects.filter(country__in=['NZ'])
<QuerySet [<Simple: 1>, <Simple: 2>]>
>>> 
# pip install django-countries --upgrade                 
Installing collected packages: django-countries
  Found existing installation: django-countries 5.0
    Uninstalling django-countries-5.0:
      Successfully uninstalled django-countries-5.0
Successfully installed django-countries-5.1.1
# ./manage.py shell                                               
>>> from countries.models import *
>>> Simple.objects.filter(country__in=['NZ'])
<QuerySet [<Simple: 1>, <Simple: 2>]>
```

**Comment by jgb** [comment]
Forgot to mention: I'm using django 1.8.19, that might make a difference?
Not using multiple=True.

**Comment by SmileyChris** [comment]
Thanks for the report, fixed in 5.2

**Comment by benjaminrigaud** [comment]
@SmileyChris thank you!
